src/components/Results/Results.tsx

```diff
@@ -363,7 +363,28 @@ export function Results() {
             />
           </Toolbar>
         </>
-      ) : undefined}
+      ) : (
```

Comment:
Instead of duplicating this part of the toolbar, you could wrap less of it on the original conditional and render these options for both.

Reply:
done in 40873a88386eba45c5f52f42bd9c97094dd719ed